src/java/org/apache/cassandra/utils/binlog/BinLogOptions.java

```diff
@@ -18,9 +18,11 @@
 
 package org.apache.cassandra.utils.binlog;
 
+import java.io.Serializable;
+
 import org.apache.commons.lang3.StringUtils;
 
-public class BinLogOptions
+public class BinLogOptions implements Serializable
```

Comment:
should remove `Serializable`, as commented in other comment, not safe to use for JMX.